### PR TITLE
Add generator file for event blocks

### LINKED ISSUE #50: Missing generators/javascript/event.js
An error shows up in both playgrounds on load.

### PR DESCRIPTION
**Pull request by rschamp**
Resolves #50

### CODE REVIEW

File: generators/javascript/event.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Generating JavaScript for math blocks.
```
Comment (rachel-fenichel) [inline generators/javascript/event.js]: Update comment.

File: generators/javascript/event.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * @author q.neutron@gmail.com (Quynh Neutron)
+ */
+'use strict';
+
+goog.provide('Blockly.JavaScript.event');
+
+goog.require('Blockly.JavaScript');
+
+Blockly.JavaScript['event_whenflagclicked'] = function(block) {
```
Comment (rachel-fenichel) [inline generators/javascript/event.js:31]: @thisandagain We should pick either camel case or underscores.  This is worst of both worlds.

File: generators/javascript/event.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Visual Blocks Language
+ *
+ * Copyright 2012 Google Inc.
```
Comment (rachel-fenichel) [inline generators/javascript/event.js]: Update year

File: generators/javascript/event.js (hunk 2 above)
Comment (rschamp) [inline generators/javascript/event.js:31]: I agree — I don't think these names are staying.

File: generators/javascript/event.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Generating JavaScript for event blocks.
+ * @author q.neutron@gmail.com (Quynh Neutron)
```
Comment (rachel-fenichel) [inline generators/javascript/event.js:23]: Nit: author

**Pull request merged**